src/js/views/bullet/bullet.js

```diff
@@ -0,0 +1,123 @@
+// React view of a bullet
+import React from "react";
+
+// https://github.com/elierotenberg/react-animate
+import Animate from '../../../../node_modules/react-animate';
```

Comment:
No need to give the full path to a node module -- just `import Animate from 'react-animate';`